sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2706,7 +2706,7 @@ object SQLConf {
       "standard directly, but their behaviors align with ANSI SQL's style")
     .version("3.0.0")
     .booleanConf
-    .createWithDefault(false)
+    .createWithDefault(sys.env.get("SPARK_ANSI_SQL_MODE").contains("true"))
```

Comment:
will `sys.env.get("SPARK_ANSI_SQL_MODE")` be null?

Reply:
The result type of `sys.env.get("SPARK_ANSI_SQL_MODE")` is Option[String]. It is totally fine here.